styles/style.css

```
@@ -0,0 +1,65 @@
+header{
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+.page-title {
+    color: rgb(89, 119, 36);
+    text-transform:uppercase;
+}
+
+.page-subtitle {
+    color: rgb(89, 119, 36);
+    text-align: center;
+    
+}
+
+ul{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    list-style: none;
+    padding-bottom: 10px;
+    padding-top: 10px;
+    font-size: 20px;
+}
+
+.link{
```

This could simply target the `a` elements themselves:

```suggestion
a {
```